mail_forward/views/compose_message.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- © 2014-2015 Grupo ESOC <www.grupoesoc.es>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
+<data>
+
+<record id="compose_form" model="ir.ui.view">
+    <field name="name">Forward Email Form</field>
+    <field name="model">mail_forward.compose_message</field>
+    <field name="inherit_id" ref="mail.email_compose_message_wizard_form"/>
+    <field name="arch" type="xml">
+        <data>
+            <!-- It's required to fill either `partner_ids` or
+                 `destination_object_id` -->
+            <xpath expr="//field[@name='partner_ids']" position="attributes">
```

Comment:
I did not avoid xpath because I prefer it. It's a standard, it's more concise, better documented and it always works.